spec/models/post_spec.rb

```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe Post, type: :model do
+  it { should validate_presence_of(:title) }
+  it { should validate_length_of(:title).is_at_most(250) }
+  it { should validate_numericality_of(:comment_counter).only_integer.is_greater_than_or_equal_to(0) }
+  it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
```

Comment:
- [ ] [Optional] [Highly recommended] Same here, It would be better professionally if you include all validation tests inside a `describe` method. 
```suggestion
describe 'validations' do
  it { should validate_presence_of(:title) }
  it { should validate_length_of(:title).is_at_most(250) }
  it { should validate_numericality_of(:comment_counter).only_integer.is_greater_than_or_equal_to(0) }
  it { should validate_numericality_of(:likes_counter).only_integer.is_greater_than_or_equal_to(0) }
end

describe 'methods' do
  -------
  -------
end
```